man/adoc/bpftrace.adoc

```diff
@@ -3793,6 +3793,15 @@ Available options:
 If the DebugInfo was rewritten by a post-linkage optimisation tool (like BOLT or AutoFDO), it might yield an incorrect address for a probe location.
 This config can force using the Symbol Table, for when the DebugInfo returns invalid addresses.
 
+**NOTE**: the following config variables are experimental (as indicated by their "unstable" prefix) so use these with caution and be aware they may be changed in future releases.
```

Comment:
Possible to use an admonition? https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#admonitions

Also maybe call out more explicitly behavior can change without warning.